electron_app/src/database/DBEmigrations/20200512111423_addBlockAllColumn.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const { Table } = require('../DBEmodel');
+
+const migrateBlockRemoteContent = async (
+  queryInterface,
+  Sequelize,
+  transaction
+) => {
+  const accountDef = await queryInterface.describeTable(Table.ACCOUNT, {
+    transaction
+  });
+
+  if (accountDef.blockAll === undefined) {
```

Comment:
shouldn't it be `accountDef. blockRemoteContent`?

Reply:
wow that was its name before, thanks!